rafs/src/metadata/direct_v6.rs

```diff
@@ -873,15 +1008,42 @@ impl RafsInode for OndiskInodeWrapper {
     /// # Safety
     /// It depends on Self::validate() to ensure valid memory layout.
     fn name(&self) -> OsString {
-        todo!()
+        if let Some(name) = self.name.take() {
+            self.name.set(Some(name.clone()));
+            name
+        } else {
+            debug_assert!(self.is_dir());
+            let cur_ino = self.ino();
+            if cur_ino == self.mapping.root_ino() {
+                return OsString::from("");
+            }
+            let parent_inode = self.mapping.inode_wrapper(self.parent()).unwrap();
+            let mut curr_name = OsString::from("");
+
+            // EROFS packs dot and dotdot, so skip them two.
+            parent_inode
+                .walk_children_inodes(
+                    2,
+                    &mut |inode: Option<Arc<dyn RafsInode>>, name: OsString, ino, offset| {
+                        if cur_ino == ino {
+                            curr_name = name;
+                            return Ok(PostWalkAction::Break);
+                        }
+                        Ok(PostWalkAction::Continue)
+                    },
+                )
+                .unwrap();
+            self.name.set(Some(curr_name.clone()));
+            curr_name
+        }
     }
-
+    // RafsV5 flags, not used by v6, return 0
     fn flags(&self) -> u64 {
-        todo!()
+        0
     }
 
     fn get_digest(&self) -> RafsDigest {
-        todo!()
+        RafsDigest { data: [0u8; 32] }
```

Comment:
RafsDigest has implemented `Default` trait, maybe we can use it directly.

Reply:
Done.